tests/test_service.py

```diff
@@ -84,6 +84,43 @@ def test_get_inventory_list(self):
         data = resp.get_json()
         self.assertEqual(len(data), 5)
 
+    def test_query_inventory_list_by_restock(self):
+        """ Query Inventories if quatity is lower than restock_level """
+        inventories = []
+        for _ in range(5):
+            test = Inventory(product_id=_, quantity=_, restock_level=3)
+            test.save()
+            inventories.append(test)
+        resp = self.app.get('/inventory',
+                            query_string='restock=true')
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), 3)
+        resp = self.app.get('/inventory',
+                            query_string='restock=false')
+
+    def test_query_inventory_list_by_restock_level(self):
+        """ Query Inventories by restock_level """
+        inventories = []
+        for _ in range(0,2):
+            test = Inventory(product_id=_, quantity=_, restock_level=20)
+            test.save()
+            inventories.append(test)
+        for _ in range(2,5):
```

Comment:
Not following pylint rules.
test_service.py:105:24: C0326: Exactly one space required after comma.